--- a/modules/bar/popouts/ClipWrapper.qml
+++ b/modules/bar/popouts/ClipWrapper.qml
@@ -23 +23,5 @@
-    x: content.isDetached ? (parent.width - content.nonAnimWidth) / 2 : 0
+    x: content.isDetached
+        ? (parent.width - content.nonAnimWidth) / 2
+        : content.bottomAttached
+            ? parent.width - content.nonAnimWidth - content.bottomRightMargin
+            : 0
@@ -26,0 +31,2 @@
+        if (content.bottomAttached)
+            return parent.height - content.nonAnimHeight - content.bottomOffset;
@@ -37,7 +42,0 @@
-    }
-
-    Behavior on x {
-        Anim {
-            duration: content.animLength
-            easing: content.animCurve
-        }
