--- a/modules/launcher/ContentList.qml
+++ b/modules/launcher/ContentList.qml
@@ -3,2 +3,3 @@
 import QtQuick
 import Caelestia.Config
+import ".."
@@ -21,1 +22,1 @@
-    readonly property bool showWallpapers: search.text.startsWith(`${GlobalConfig.launcher.actionPrefix}wallpaper `)
+    readonly property bool showWallpapers: search.text.startsWith(`${CortetsuConfig.actionPrefix}wallpaper `)
--- a/modules/launcher/items/CalcItem.qml
+++ b/modules/launcher/items/CalcItem.qml
@@ -4,2 +4,3 @@
 import Caelestia
 import Caelestia.Config
+import "../.."
@@ -13,1 +14,1 @@
-    readonly property string math: list.search.text.slice(`${GlobalConfig.launcher.actionPrefix}calc `.length)
+    readonly property string math: list.search.text.slice(`${CortetsuConfig.actionPrefix}calc `.length)
@@ -81,1 +82,1 @@
-                    Quickshell.execDetached([...GlobalConfig.general.apps.terminal, "fish", "-C", `exec qalc -i '${root.math}'`]);
+                    Quickshell.execDetached([...CortetsuConfig.terminalCommand, "fish", "-C", `exec qalc -i '${root.math}'`]);
