--- a/modules/launcher/AppList.qml
+++ b/modules/launcher/AppList.qml
@@ -10,2 +10,3 @@
 import qs.utils
 import qs.modules.launcher.services
+import ".."
@@ -49,1 +50,1 @@
-        const prefix = GlobalConfig.launcher.actionPrefix;
+        const prefix = CortetsuConfig.actionPrefix;
@@ -137,1 +138,1 @@
-        const apps = GlobalConfig.launcher.favouriteApps;
+        const apps = CortetsuConfig.favouriteApps;
@@ -140,1 +141,1 @@
-            GlobalConfig.launcher.favouriteApps = apps.filter(a => a !== id);
+            CortetsuConfig.setFavouriteApps(apps.filter(a => a !== id));
@@ -142,1 +143,1 @@
-            GlobalConfig.launcher.favouriteApps = [...apps, id];
+            CortetsuConfig.setFavouriteApps([...apps, id]);
@@ -221,1 +222,1 @@
-                    GlobalConfig.launcher.favouriteApps,
+                    CortetsuConfig.favouriteApps,
@@ -399,3 +400,3 @@
             readonly property string math:
                 root.search.text.slice(
-                    `${GlobalConfig.launcher.actionPrefix}calc `.length
+                    `${CortetsuConfig.actionPrefix}calc `.length
@@ -489,1 +490,1 @@
-                            ...GlobalConfig.general.apps.terminal,
+                            ...CortetsuConfig.terminalCommand,
--- a/modules/launcher/services/Apps.qml
+++ b/modules/launcher/services/Apps.qml
@@ -4,2 +4,3 @@
 import Caelestia.Models
 import qs.utils
+import "../.."
@@ -57,1 +58,1 @@
-            root.launchInScope(entry, [...GlobalConfig.general.apps.terminal, `${Quickshell.shellDir}/assets/wrap_term_launch.sh`, ...entry.command]);
+            root.launchInScope(entry, [...CortetsuConfig.terminalCommand, `${Quickshell.shellDir}/assets/wrap_term_launch.sh`, ...entry.command]);
@@ -63,1 +64,1 @@
-        const prefix = GlobalConfig.launcher.specialPrefix;
+        const prefix = CortetsuConfig.specialPrefix;
@@ -104,1 +105,1 @@
-    useFuzzy: GlobalConfig.launcher.useFuzzy.apps
+    useFuzzy: CortetsuConfig.useFuzzyApps
@@ -110,2 +111,2 @@
-        favouriteApps: GlobalConfig.launcher.favouriteApps
-        entries: DesktopEntries.applications.values.filter(a => !Strings.testRegexList(GlobalConfig.launcher.hiddenApps, a.id))
+        favouriteApps: CortetsuConfig.favouriteApps
+        entries: DesktopEntries.applications.values.filter(a => !Strings.testRegexList(CortetsuConfig.hiddenApps, a.id))
