--- a/modules/background/Background.qml
+++ b/modules/background/Background.qml
@@ -7,0 +8 @@
+import qs.modules
@@ -11 +12 @@
-    model: Screens.screens.filter(s => GlobalConfig.forScreen(s.name).background.enabled)
+    model: CortetsuScreens.screens
@@ -22,2 +23,2 @@
-        WlrLayershell.layer: contentItem.Config.background.wallpaperEnabled ? WlrLayer.Background : WlrLayer.Bottom
-        color: contentItem.Config.background.wallpaperEnabled ? "black" : "transparent"
+        WlrLayershell.layer: CortetsuConfig.wallpaperEnabled ? WlrLayer.Background : WlrLayer.Bottom
+        color: CortetsuConfig.wallpaperEnabled ? "black" : "transparent"
@@ -49 +50 @@
-                active: Config.background.wallpaperEnabled
+                active: CortetsuConfig.wallpaperEnabled
@@ -65 +66 @@
-            active: Config.background.desktopClock.enabled
+            active: CortetsuConfig.desktopClockEnabled
@@ -68 +69 @@
-            anchors.leftMargin: Tokens.padding.extraLargeIncreased + Tokens.sizes.bar.innerWidth + Math.max(Tokens.padding.small, Config.border.thickness)
+            anchors.leftMargin: Tokens.padding.extraLargeIncreased + Tokens.sizes.bar.innerWidth + Math.max(Tokens.padding.small, CortetsuConfig.borderThickness)
@@ -70 +71 @@
-            state: Config.background.desktopClock.position
+            state: CortetsuConfig.desktopClockPosition
