--- a/modules/utilities/Wrapper.qml
+++ b/modules/utilities/Wrapper.qml
@@ -8 +7,0 @@
-import qs.modules.sidebar as Sidebar
@@ -15 +13,0 @@
-    required property Sidebar.Wrapper sidebar
@@ -17 +14,0 @@
-    property real horizontalStretch
@@ -27 +24 @@
-    readonly property bool shouldBeActive: screenState.sidebar || (screenState.utilities && Config.utilities.enabled && !(screenState.session && Config.session.enabled))
+    readonly property bool shouldBeActive: screenState.utilities && Config.utilities.enabled && !(screenState.session && Config.session.enabled)
@@ -31 +27,0 @@
-    property real sidebarLerp
@@ -34 +30 @@
-    anchors.bottomMargin: (-implicitHeight - 5) * offsetScale
+    anchors.bottomMargin: 66 + (-implicitHeight - 5 - 66) * offsetScale
@@ -36 +32 @@
-    implicitWidth: sidebar.width * (1 - sidebar.offsetScale) * horizontalStretch * sidebarLerp + Tokens.sizes.utilities.width * (1 - sidebarLerp)
+    implicitWidth: Math.min(Tokens.sizes.utilities.width, parent.width - 16)
@@ -38,30 +33,0 @@
-
-    states: State {
-        name: "attachedToSidebar"
-        when: root.screenState.sidebar
-
-        PropertyChanges {
-            root.sidebarLerp: 1
-        }
-    }
-
-    transitions: [
-        Transition {
-            from: ""
-
-            Anim {
-                property: "sidebarLerp"
-                duration: Tokens.anim.durations.expressiveDefaultSpatial / 2
-                easing: Tokens.anim.standardAccel
-            }
-        },
-        Transition {
-            to: ""
-
-            Anim {
-                property: "sidebarLerp"
-                duration: Tokens.anim.durations.expressiveDefaultSpatial / 2
-                easing: Tokens.anim.standardDecel
-            }
-        }
-    ]
