--- a/modules/launcher/Content.qml
+++ b/modules/launcher/Content.qml
@@ -5,1 +5,2 @@
 import Caelestia.Config
+import ".."
@@ -51,1 +52,1 @@
-        placeholderText: qsTr("Type \"%1\" for commands").arg(GlobalConfig.launcher.actionPrefix)
+        placeholderText: qsTr("Type \"%1\" for commands").arg(CortetsuConfig.actionPrefix)
@@ -62,2 +63,2 @@
-                } else if (text.startsWith(GlobalConfig.launcher.actionPrefix)) {
-                    if (text.startsWith(`${GlobalConfig.launcher.actionPrefix}calc `))
+                } else if (text.startsWith(CortetsuConfig.actionPrefix)) {
+                    if (text.startsWith(`${CortetsuConfig.actionPrefix}calc `))
@@ -109,1 +110,1 @@
-            if (GlobalConfig.launcher.vimKeybinds &&
+            if (CortetsuConfig.vimKeybinds &&
--- a/modules/launcher/items/AppItem.qml
+++ b/modules/launcher/items/AppItem.qml
@@ -4,1 +4,2 @@
 import Caelestia.Config
+import "../.."
@@ -80,1 +81,1 @@
-            active: root.modelData && Strings.testRegexList(GlobalConfig.launcher.favouriteApps, root.modelData.id)
+            active: root.modelData && Strings.testRegexList(CortetsuConfig.favouriteApps, root.modelData.id)
