--- a/modules/drawers/ContentWindow.qml
+++ b/modules/drawers/ContentWindow.qml
@@ -158,1 +158,1 @@
-        opacity: root.screenState.overview ? 0.58 : ((root.screenState.session && Config.session.enabled) || panels.popouts.detachedMode !== "" ? 0.5 : 0)
+        opacity: root.screenState.cortetsuState?.overview ? 0.58 : ((root.screenState.session && Config.session.enabled) || panels.popouts.detachedMode !== "" ? 0.5 : 0)
