--- a/services/Wallpapers.qml
+++ b/services/Wallpapers.qml
@@ -7,1 +7,2 @@
 import qs.services
+import "../modules"
@@ -15,1 +16,1 @@
-    readonly property list<string> smartArg: GlobalConfig.services.smartScheme ? [] : ["--no-smart"]
+    readonly property list<string> smartArg: CortetsuConfig.smartScheme ? [] : ["--no-smart"]
@@ -91,1 +92,1 @@
-    useFuzzy: GlobalConfig.launcher.useFuzzy.wallpapers
+    useFuzzy: CortetsuConfig.useFuzzyWallpapers
--- a/utils/Paths.qml
+++ b/utils/Paths.qml
@@ -5,1 +5,2 @@
 import Caelestia.Config
+import "../modules"
@@ -22,1 +23,1 @@
-    readonly property string wallsdir: Quickshell.env("CAELESTIA_WALLPAPERS_DIR") || absolutePath(GlobalConfig.paths.wallpaperDir)
+    readonly property string wallsdir: Quickshell.env("CORTETSU_WALLPAPERS_DIR") || absolutePath(CortetsuConfig.wallpaperDirectory)
