--- a/shell.qml
+++ b/shell.qml
@@ -19 +19 @@
-    settings.watchFiles: true
+    settings.watchFiles: false
@@ -31,0 +32 @@
+    BottomHub {}
@@ -37,6 +39,12 @@
     ConfigToasts {}
     Shortcuts {}
     BatteryMonitor {}
+    OverviewController {}
+    CalendarController {}
+    ClipboardController {}
+    HardwareController {}
+    DisplayController {}
+    WallpaperController {}
     IdleMonitors {
         lock: lock
     }
